File: Service/ClientCare/RMA.Service.ClientCare.Contracts/Entities/Member/LetterOfGoodStanding.cs
namespace RMA.Service.ClientCare.Contracts.Entities.Member
{
    public class LetterOfGoodStanding
    {
        public int LetterOfGoodStandingId { get; set; }
        public int RolePlayerId { get; set; }
        public System.DateTime IssueDate { get; set; }
        public System.DateTime ExpiryDate { get; set; }
        public string CertificateNo { get; set; }
        public string CreatedBy { get; set; }
        public System.DateTime CreatedDate { get; set; }
        public string ModifiedBy { get; set; }
        public System.DateTime ModifiedDate { get; set; }

        // properties not in database
        public string MemberName { get; set; }
        public string MemberEmail { get; set; }
        public int PolicyId { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/clientcare/src/app/policy-manager/shared/entities/letter-of-good-standing.ts
export class LetterOfGoodStanding {
  letterOfGoodStandingId: number;
  rolePlayerId: number;
  issueDate: Date;
  expiryDate: Date;
  certificateNo: string;
  createdBy: string;
  createdDate: Date;
  modifiedBy: string;
  modifiedDate: Date;

  // properties not in database
  memberName: string;
  memberEmail: string;
  policyId: number;
}
